composer.json

```diff
@@ -39,11 +39,11 @@
         "linkorb/jsmin-php": "~1.0",
         "wikimedia/less.php": "~3.0",
         "scssphp/scssphp": "~1.0",
-        "symfony/yaml": "^3.4",
+        "symfony/yaml": "^5.1",
         "twig/twig": "~2.0",
         "league/csv": "~9.1",
         "nesbot/carbon": "^2.0",
-        "laravel/framework": "~6.0",
+        "laravel/framework": "^8.0",
```

Comment:
For whatever reason my composer refused to perform updates to 8.0 without the ^. -vvv gave no meaningful insights. 
I tried this and it worked, I can change it to ~ if you want.

Reply:
can we target dev-master or is that going to be too much of a headache?

Reply:
I would keep this at 8 for now for the work that's occurring so that if the workflow comes that you can do a version release before they release Laravel 9, you can continue on and release 1.2 without being dependent on "in flux" code, and then target 9 in 1.3.

But that's up to you/the rest of the core team to decide, you have a better overview of the general development progress towards a new release 1.2 and if it will be released before/after laravel 9 hits the market.

Reply:
I changed the targetting to dev master. For tinker I needed to target dev-develop specifically to get it working. When they merge that in their master we can target dev-master there.
All unit tests on our side pass.